src/manager.cpp

```diff
@@ -233,6 +283,46 @@ sdbusplus::async::task<bool>
 
         co_return false;
     }
+    else if (dataSyncCfg._notifySibling.has_value())
+    {
+        if (dataSyncCfg._notifySibling.value()._paths.has_value())
```

Comment:
Move all this logic into triggerSiblingNotification, since that function is responsible for triggering the notification. The decision of whether a notification is required and whether it succeeds should both be handled within that function.